<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMahjonggLayoutSelector</class>
 <widget class="QWidget" name="KMahjonggLayoutSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>461</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="kcfg_RandomLayout">
       <property name="text">
        <string>Random Layout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetNoRandom" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="layoutNoRandom">
          <item>
           <layout class="QHBoxLayout" name="_2">
            <item>
             <layout class="QVBoxLayout" name="_3">
              <item>
               <widget class="QListWidget" name="layoutList">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="getNewButton">
                <property name="text">
                 <string>Download New Layoutsâ€¦</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Preview</string>
              </property>
              <layout class="QVBoxLayout" name="_4">
               <item>
                <widget class="QLabel" name="layoutPreview">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>240</width>
                   <height>180</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Layout Details</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelAuthor">
               <property name="text">
                <string>Author:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="layoutAuthor"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelContact">
               <property name="text">
                <string>Contact:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="layoutContact"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelDescription">
               <property name="text">
                <string>Description:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="layoutDescription"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="kcfg_Layout">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_RandomLayout</sender>
   <signal>toggled(bool)</signal>
   <receiver>KMahjonggLayoutSelector</receiver>
   <slot>useRandomLayoutToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>useRandomLayoutToggled(bool)</slot>
 </slots>
</ui>
